### ISSUE TITLE
Individual project pages' Wiki and Overview should not have alt="/"

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2626]
### Overview
As a developer, we need to make sure the site follows Web Content Accessibility Guidelines (WCAG). For this issue, we will fix the Wiki and Overview images' attribute `alt="/"` on the individual project pages' resources section. Examples of pages with this problem: [Civic Tech Index](https://www.hackforla.org/projects/civic-tech-index), [100 Automations](https://www.hackforla.org/projects/100-automations), [311 Data](https://www.hackforla.org/projects/311-data).

<details>
<summary>Example of Wiki and Overview images with alt="/" on the Civic Tech Index's project page</summary>

<img width="673" alt="altEqualSlash" src="https://user-images.githubusercontent.com/31293603/147394105-8b186d19-2768-469e-93e4-8bf4ecc4f283.png">

</details>

### Action Items
- [x] Research and decide what the alt text, if any, should be
  - [x] If no alt text is needed, then it should be `alt=""`
- [x] Fix the alt text attribute

### Resources
- [_layouts/project.html](https://github.com/hackforla/website/blob/d3b10fadfefa646ade8bdd5e3de6b89fa1ad466b/_layouts/project.html)
- [_includes/resource-card.html](https://github.com/hackforla/website/blob/d3b10fadfefa646ade8bdd5e3de6b89fa1ad466b/_includes/resource-card.html)
- [An alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [Alt text no more than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
- [Using alt attributes on img elements](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Logo image within link text](https://www.w3.org/WAI/tutorials/images/functional/#example-2-logo-image-within-link-text)

**Comment by arpitapandya** [comment]
Availability : 6 Hrs
ETA: EOD 2022-07-28

**Comment by Unity7** [comment]
Availability: 4 Hours
ETA: 10/1/22

**Comment by Unity7** [comment]
Availability: 4 Hours
ETA: 10/8/22
Blockers: Unable to render img files with alt text attribute in project.html. Might need a better understanding of the stack that is being used to render the md file.

Currently the image src is being pulled from the md file but not able to attach the alt text along with the img being rendered through a variable.

**Comment by Unity7** [comment]
Progress: Received guidance, resources, and direction to make the next steps
Blockers: Need better understanding of jekyll
Availability: 4
ETA: 10/19/22

**Comment by Unity7** [comment]
Progress: Reading documentation on Jekyll and reviewing a ticket with a similar issue as reference
Blockers: Need more time to go over documentation
Availability: 4
ETA: 10/23/22

**Comment by AzaniaBG** [comment]
@JessicaLucindaCheng ~ I'm reviewing the PR for this issue and came across [this resource](https://www.w3.org/WAI/tutorials/images/functional/#example-2-logo-image-within-link-text) for using alt text with a logo. Would it be okay to reference this as a resource in addition to the resources already provided?

**Comment by JessicaLucindaCheng** [comment]
@AzaniaBG 

> @JessicaLucindaCheng ~ I'm reviewing the PR for this issue and came across [this resource](https://www.w3.org/WAI/tutorials/images/functional/#example-2-logo-image-within-link-text) for using alt text with a logo. Would it be okay to reference this as a resource in addition to the resources already provided?

Thanks for reviewing the PR! Sure, I added it to the Resources section above.

**Comment by ExperimentsInHonesty** [comment]
@Unity7 Hi, it turns out your issue has a larger fix needed.  Basically the resources cards should have specific alt text based on which ones they are

So for instance 
- If its GitHub or Slack, then the alt text is `alt=""` for all the individual project pages
- If its the Hack for LA Getting Started Guide then `alt="Hack for LA` for all the individual project pages
- If its the wiki, site or overview, then it's `alt="Project"` for all the individual project pages

**Comment by Unity7** [comment]
> If its GitHub or Slack, then the alt text is alt="" for all the individual project pages
--For GitHub and Slack the code is using an SVG and we aren't able to associate alt text attribute with SVG since it uses a title tag instead. Nothing is to be done for this one.

>If its the Hack for LA Getting Started Guide then alt="Hack for LA for all the individual project pages
--For Hack for Lack Getting started guide the code is using an SVG and we aren't able to associate alt text attribute with SVG since it uses a title tag instead. Nothing is to be done for this one.

>If its the wiki, site or overview, then it's alt="Project" for all the individual project pages
--it sounds like this is the only portion that renders an img and your comments suggest to hard code the alt text for all img to be "Project" making them static.  alt='Project'

**Comment by ExperimentsInHonesty** [comment]
Can we use an ARIA tag for these? 

> If its GitHub or Slack, then the alt text is alt="" for all the individual project pages
> --For GitHub and Slack the code is using an SVG and we aren't able to associate alt text attribute with SVG since it uses a title tag instead. Nothing is to be done for this one.
 
> If its the Hack for LA Getting Started Guide then alt="Hack for LA for all the individual project pages
> --For Hack for Lack Getting started guide the code is using an SVG and we aren't able to associate alt text attribute with SVG since it uses a title tag instead. Nothing is to be done for this one.

If yes, let's make a new issue.
If no, add a note to this issue and put back into the QA column

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 

1) No, I don't think ARIA labels are the best way to do this. I think it's better to put the svg in img tag and use an alt property with it. Example: 
  `<img class="header-hero-image" src="/assets/images/toolkit/toolkit-hero.svg" alt="" >`
2) Please look over the issue again, and see if anything else on the project detail page needs to be fixed to be WCAG compliant. 

> Can we use an ARIA tag for these?
> 
> > If its GitHub or Slack, then the alt text is alt="" for all the individual project pages
> > --For GitHub and Slack the code is using an SVG and we aren't able to associate alt text attribute with SVG since it uses a title tag instead. Nothing is to be done for this one.
> 
> > If its the Hack for LA Getting Started Guide then alt="Hack for LA for all the individual project pages
> > --For Hack for Lack Getting started guide the code is using an SVG and we aren't able to associate alt text attribute with SVG since it uses a title tag instead. Nothing is to be done for this one.
> 
> If yes, let's make a new issue. If no, add a note to this issue and put back into the QA column

**Comment by ExperimentsInHonesty** [comment]
I agree this issue is complete.  @hackforla/website-merge Please make an issue for making each of the SVGs on the page have `alt=""`  Which will require wrapping them in an `<img src="...`

**Comment by ExperimentsInHonesty** [comment]
- Created an ER https://github.com/hackforla/website/issues/4242

**Comment by JessicaLucindaCheng** [comment]
Here is a wiki page for how to make a SVG WCAG compliant: https://github.com/hackforla/website/wiki/How-to-make-SVGs-and-other-images-WCAG-compliant
